src/ui/view/stakeholder.rs

```diff
@@ -64,114 +66,129 @@ impl StakeholderACKFundsView {
+                                    ctx.converter.converts(*activating_balance)
+                                )))
+                                .size(20),
+                            )
+                            .push(text::simple(&format!(
+                                " {} are waiting for other stakeholders' approval",
+                                ctx.converter.unit
+                            )))
+                            .align_items(Align::Center),
+                    ),
+            );
+        if vaults.len() > 0 {
+            col = col.push(Container::new(
+                Column::new()
+                    .push(text::simple(" Click on the vaults to delegate:"))
+                    .push(Column::with_children(vaults).spacing(5))
+                    .spacing(20),
+            ))
         } else {
-            vaults_header = vaults_header
-                .push(
-                    button::primary(
-                        &mut self.secured_vaults_button,
-                        button::button_content(None, "Available vaults"),
-                    )
-                    .on_press(Message::FilterVaults(
-                        VaultFilterMessage::Status(&VaultStatus::INACTIVE),
-                    )),
-                )
-                .push(
-                    button::transparent(
-                        &mut self.active_vaults_button,
-                        button::button_content(None, "Delegated vaults"),
-                    )
-                    .on_press(Message::FilterVaults(
-                        VaultFilterMessage::Status(&VaultStatus::ACTIVE),
-                    )),
-                );
+            col = col.push(
+                Container::new(text::simple("No more funds can be delegated to managers"))
+                    .padding(5),
+            )
         }
-        let col = Column::new()
+
+        let modal = Column::new()
             .push(
-                card::white(Container::new(
-                    Column::new()
-                        .push(
-                            Row::new()
-                                .push(text::bold(text::simple(&format!(
-                                    "{}",
-                                    ctx.converter.converts(*active_balance)
-                                ))))
-                                .push(text::simple(&ctx.converter.unit.to_string()))
-                                .spacing(10)
-                                .align_items(Align::Center),
+                Row::new()
+                    .push(
+                        Container::new(
+                            Tooltip::new(
+                                Row::new()
+                                    .push(icon::tooltip_icon().size(15))
+                                    .push(text::small(" Help")),
+                                "By delegating, you give rights to managers to move funds, \n but you still have control on it.",
```

Comment:
nit: "By delegating you allow managers to spend the funds,\n but you can still revert any undesired transaction."